CODEOWNERS

```diff
@@ -87,3 +87,5 @@ extensions/filters/common/original_src @snowp @klarose
 /*/extensions/filters/network/ext_authz @gsagula @dio
 /*/extensions/filters/network/tcp_proxy @alyssawilk @zuercher
 /*/extensions/filters/network/echo @htuch @alyssawilk
+# support for on-demand VHDS requests
+/*/extensions/filters/http/on_demand @dmitri-d
```

Comment:
Please tag me and @lambdai as CODEOWNERS here as well, as we will be involved in long term maintenance as well.

Reply:
done